components/server/package.json

```diff
@@ -7,10 +7,9 @@
     "start": "node ./dist/main.js",
     "start-inspect": "node --inspect=0.0.0.0:9229 ./dist/main.js",
     "generate": "leeway run components/spicedb:generate-ts > src/authorization/definitions.ts && npx prettier --write src/authorization/definitions.ts",
-    "build": "yarn generate && yarn lint && npx tsc",
+    "build": "yarn clean && yarn generate && yarn lint && npx tsc",
     "lint": "yarn eslint src/*.ts src/**/*.ts",
     "lint:fix": "yarn eslint src/*.ts src/**/*.ts --fix",
-    "build:clean": "yarn clean && yarn build",
```

Comment:
not cleaning on yarn build doesn't make it faster but would leave *.js files in `/dist` when e.g. switching between branches.